src/gui/AstroCalcDialog.cpp

```diff
@@ -6210,7 +6522,8 @@ void AstroCalcDialog::changeEclipsesTab(int index)
 	const QMap<int, QString> headermap = {
 		{0,	q_("Table of solar eclipses")},
 		{1,	q_("Table of solar eclipses visible in current location")},
-		{2,	q_("Table of lunar eclipses")}
+		{2,	q_("Table of lunar eclipses")},
+		{3,	q_("Transits of Mercury and Venus")}
```

Comment:
Maybe "Transits of Mercury and Venus across the Sun" will be better here?

Reply:
Agree.